### PR TITLE
[#1791] Match Provider model with PC provider profile

### LINKED ISSUE #1791: Match marketplace Provider model with PC provider profile
- [x] Add missisng vocabularies
- [x] Add missing fields to the Provider model:
  - abbreviation - string
  - website - string
  - legalEntity - bool
  - legalStatus - Vocabulary
  - description - string
  - logo
  - multimedia - string[]
  - ScientificDomains - ScientificDomain[]
  - tags - string[]
  - location:
    - streetNameAndNumber - string
    - postalCode - string
    - city - string
    - region - string
    - country -  Country
  - mainContact - MainContact
  - publicContacts - PublicContacts
  - lifeCycleStatus - Vocabulary
  - certifications - string[]
  - hostingLegalEntity - string
  - participationCountries - Country[]
  - affiliations - string[]
  - networks - Vocabulary[]
  - structureTypes - Vocabulary[]
  - esfriDomains - Vocabulary[]
  - esfriType - Vocabulary
  - merilScientificDomains - Vocabulary[]
  - areasOfActivity - Vocabulary[]
  - societalGrandChallenges - Vocabulary[]
  - nationalRoadmaps - string[]
- [x] Add fields to form
- [x] Add missing fields to the importers

### PR DESCRIPTION
**Pull request by goreck888**
- Add provider_vocabularies
- Add new models for new vocabularies:
  - area_of_activity
  - esfri_domain
  - esfri_type
  - legal_status
  - meril_scientific_domain
  - network
  - societal_grand_challenge
  - structure_type
- Add data to populate vocabularies
- Add new fields to Provider model

Fixes #1791

### CODE REVIEW

File: app/javascript/app/controllers/form_controller.js [hunk 1]
```diff
@@ -5,14 +5,15 @@ export default class extends Controller {
...
-    "relatedPlatforms", "fixme"]
+    "relatedPlatforms", "affiliations", "national_roadmaps", "fixme"]
 
   initialize() {
     this.addListenersForCollapse();
     this.disableFormButtons();
   }
 
   onScroll(event) {
+    console.log("NAKURWIAM")
```
Comment (kmarszalek) [inline app/javascript/app/controllers/form_controller.js:16]: seriously?

File: app/models/concerns/logo_attachable.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module LogoAttachable
+  def logo_variable
+    if logo.present? && !logo.variable?
+      errors.add(:logo, "^Sorry, but the logo format you were trying to attach is not supported
```
Comment (kmarszalek) [inline app/models/concerns/logo_attachable.rb:6]: o would change "Sorry, but the logo.." to "The logo ..." here and in service model.

Comment (bwilk) [reply app/models/concerns/logo_attachable.rb:6]: The logo format you're trying to attach is not supported.

Comment (bwilk) [reply app/models/concerns/logo_attachable.rb:6]: Please use the text above

**Review by michal-szostak — APPROVED** [review]
Looks generally alright

**Review by kmarszalek — APPROVED** [review]

**Pull request merged**